array-methods.js

```diff
@@ -1,12 +1,24 @@
+'use strict';
 var dataset = require('./dataset.json');
 
 /*
   create an array with accounts from bankBalances that are
   greater than 100000
   assign the resulting new array to `hundredThousandairs`
 */
+
+
 var hundredThousandairs = null;
 
+function findOverHundredK(arr) {
+  if (arr.amount > 100000) {
+    return arr;
+  }
+}
+
+hundredThousandairs = dataset.bankBalances.filter(findOverHundredK);
```

Comment:
Try to inline these functions since they are small enough. Inline functions are going to be very common in the future lectures and exercises.